tests/fixtures.py

```diff
@@ -62,6 +65,9 @@ def session(db, engine, request):
     # Patch the global session in hydra_base
     hydra_base.db.DBSession = session
 
+    if six.PY2 and isinstance(session.connection().connection.connection,sqlite3.Connection):
+        session.connection().connection.connection.text_factory = lambda x: unicode(x, 'utf-8', 'ignore')
```

Comment:
`session.connection().connection.connection` ... OK I won't ask why this works!

Reply:
session.connection()  ==> sqlalchemy connection object
session.connection().connection ==> the connection fairy
session.connection().connection.connection ==> the actual sqlite connection